<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="InstallLayout">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Install theOS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Give theOS a permanant place on your PC. Install theOS on your hard drive, either replacing another operating system or next to another operating system.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Open theOS Installer</string>
          </property>
          <property name="icon">
           <iconset theme="install_theos">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Start typing to search, run a command, or open a web address</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Exit</string>
         </property>
         <property name="icon">
          <iconset theme="system-shutdown">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="offFrame">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>You're about to power off your PC. Are you sure?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="text">
         <string>Power Off</string>
        </property>
        <property name="icon">
         <iconset theme="system-shutdown">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Turns off your computer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_2">
        <property name="text">
         <string>Reboot</string>
        </property>
        <property name="icon">
         <iconset theme="system-reboot">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Turns off your computer and turns it back on again</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_3">
        <property name="text">
         <string>Log Out</string>
        </property>
        <property name="icon">
         <iconset theme="system-log-out">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Ends your session and keeps the computer on for other people to use</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_6">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Lock</string>
        </property>
        <property name="icon">
         <iconset theme="system-lock-screen">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Locks your workspace with your password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Suspend</string>
        </property>
        <property name="icon">
         <iconset theme="system-suspend">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Puts your computer in a low power state so that it opens up quickly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Switch Users</string>
        </property>
        <property name="icon">
         <iconset theme="system-switch-user">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Switches to the logon screen and keeps your session in the background</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="commandLinkButton_7">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Hibernate</string>
        </property>
        <property name="icon">
         <iconset theme="system-suspend-hibernate">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="description">
         <string>Powers off your computer, but when powered back on, restores the session.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset theme="dialog-cancel">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
